holocron_api/api/models/standard.py

```diff
@@ -21,23 +35,41 @@ def __str__(self):
         return self.campaign_key
 
 
+class Tactic(models.Model):
+    tactic_id = models.AutoField(primary_key=True)
+    tactic_key = models.CharField(max_length=100, blank=True, null=True,
```

Comment:
After some thought, i'm not sure how I feel about leaving the "key" in here for all time... considering that it was our sort of learning mishap... We're basically forcing this to stay in the system forever because we hadn't figured out what we wanted... I wonder if we should find a workaround for our old reporting... or maybe add some kind of IGLU script to convert the old ones with "keys" to IDs when it sees them... Should we add a bug for this and try to solve it with IGLU?